resources/assets/less/colours.less

```diff
@@ -58,11 +58,10 @@
 @community_colour: @blue;
 @store_colour: @pink;
 
-// https://about.twitter.com/press/brand-assets
-@twitter-colour: #55acee;
-
-// https://www.facebookbrand.com
+// https://brandcolors.net
```

Comment:
the site doesn't like to primary source? :(

Also I can't find where soundcloud color guide is located. Good amount of their styling use `#f50` though.